examples/lib/stories/bridge_libraries/flame_forge2d/animated_body_example.dart

```diff
@@ -32,29 +33,29 @@ class AnimatedBodyExample extends Forge2DGame with TapDetector {
     );
 
     final boundaries = createBoundaries(this);
-    boundaries.forEach(add);
+    world.addAll(boundaries);
   }
 
   @override
   void onTapDown(TapDownInfo info) {
     super.onTapDown(info);
-    final position = info.eventPosition.game;
+    final position = screenToWorld(info.eventPosition.widget);
```

Comment:
should this transformation be handled in the TapCallbacks mixin?

Reply:
That would only work in the case of a single camera, later when there is a default camera we could probably do that though, and provide the screen position too if the user wants to project with another camera.

Reply:
Fixed!